### PR TITLE
- fix getData issue when there are multiple groups

### PR DESCRIPTION
**Pull request by mohmdrezk**
- fix the group(i).subNumber when there are multiple groups
This is related to an already open issue. 

Screen Shot 2019-10-10 at 14 57 15 2

### CODE REVIEW

**Comment by mohmdrezk** [comment]
Related to the open issue #22

File: subfun/getData.m [hunk 1]
```diff
@@ -2,9 +2,6 @@
 % The following structure will be the base that the function will use to run
 % the preprocessing pipeline according to the BIDS Structure
 
-% we check the option inputs and add any missing field with some defaults
-opt = checkOptions(opt);
```
Comment (Remi-Gau) [inline subfun/getData.m]: I would not remove this sub-function: it was created to make sure to add any missing fields in the `opt` structure that the user might have forgotten to specify

**Comment by Remi-Gau** [comment]
OK I am going I think that we need to create some unit tests to make sure that our code does what we want. I will work on this and push that on this pull request.

If you want to know more:
- https://the-turing-way.netlify.com/testing/testing.html
- https://www.mathworks.com/help/matlab/matlab-unit-test-framework.html

File: subfun/getData.m (hunk 1 above)
Comment (mohmdrezk) [reply subfun/getData.m]: That was probably a mistake from my side. I was just clearing spaces and probably removed it by mistake. 
The intended part is in the loop at the bottom of the script.

**Comment by mohmdrezk** [comment]
Great. Looking forward it. 
Come on, no fault. Thanks to you I am pulling, pushing, and forking stuff :)

**Pull request merged**